tags/terminal/unix_shell.py

```diff
@@ -6,21 +6,23 @@
 tag: user.generic_unix_shell
 """
 
+mod.list("ls_args", "argument mapping for ls command")
+
+ctx.lists["user.ls_args"] = {
+    'list': '-l',
+    'all': '-a',
+}
+
+@mod.action_class
+class JoinAction:
+    def join_strings(l: list, s: str):
+        """Joins the strings using the given separator"""
+        return s.join(l)
```

Comment:
This probably shouldn't go in this file but I don't know where to put it right now.